pkgs/applications/networking/instant-messengers/teams/default.nix

```diff
@@ -38,6 +39,7 @@ stdenv.mkDerivation rec {
 
   preFixup = ''
     gappsWrapperArgs+=(--prefix PATH : "${coreutils}/bin:${gawk}/bin:${xdg-utils}/bin")
+    gappsWrapperArgs+=(--prefix LD_LIBRARY_PATH : "${libappindicator-gtk3}/lib")
```

Comment:
Would this work if this was added to `runtimeDependencies` instead? Pulseaudio and udev use that mechanism already.

Reply:
That seems a better idea, changed.